webui/react/src/pages/WorkspaceDetails/WorkspaceProjects.tsx

```diff
@@ -308,7 +308,7 @@ const WorkspaceProjects: React.FC<Props> = ({ workspace, id, pageRef }) => {
         {children}
       </ProjectActionDropdown>
     ),
-    [fetchProjects, user, workspace?.archived],
+    [user, workspace?.archived],
```

Comment:
I personally wanna avoid using `// eslint-disable-next-line react-hooks/exhaustive-deps`, but it seems like it's the easiest way to fix this issue, so adding the `// eslint-disable-next-line react-hooks/exhaustive-deps` would be helpful for ppl who gonna work on this file not to add dependency accidentally.

Reply:
Updated!